objectives/results/_add_sub_stat.py

```diff
@@ -5,16 +5,13 @@
 from objectives.results._apply_characters_party import ApplyToCharacter, ApplyToCharacters
 
 def AddStat(stat_address):
+    MAX_STAT = 0x80
     return [
         asm.LDA(stat_address, asm.ABS_Y),
-        asm.CLC(),
```

Comment:
Is this a safe assumption? Where does the last carry clear occur?

Reply:
After a closer look I found that my assumptions here were incorrect. I readded the CLC.